docs/README.md

```diff
@@ -46,6 +46,42 @@ The web-server will setup everything and connect itself to a postgis database, a
 
 Remember to recomment the line `python manage.py migrate --settings=local_settings` in the `entrypoint.sh` file.
 
+## Accessing the local server
+
+After setting up the AIS with Docker, you can access it in a web browser with the address "localhost:3000"
+
+### Creating a super user account
+
+You will notice that there is no way to login to the system. This is because there are no users in the newly created, empty database. To solve this, you have to enter the Docker container and create a super user, by following the steps below.
+
+1. First list the Docker processes using the command ```docker ps```. Find the process for the ais-web Docker container and note down the value in field "CONTAINER ID".
+
+2. Then, you want to execute an interactive ```sh``` shell on the ais-web container, with the following command:
+
+    ```docker exec -it containerid sh```
```

Comment:
(Just some info don't have to fix): When doing block-selections that shouldn't take up the entire line, just using a single ´ around the text will suffice.
Tripple backticks are only required when you want results like this:
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/30767818/234957609-e45375da-599e-4ab7-a747-856976e64d1e.png">